src/Sarif.Converters/HdfFile.cs

```diff
@@ -0,0 +1,576 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+
+using Newtonsoft.Json;
+using Newtonsoft.Json.Converters;
+
+namespace Microsoft.CodeAnalysis.Sarif.Converters
+{
+    // <auto-generated />
+    // To parse this JSON data use var hdfFile = HdfFile.FromJson(jsonString)
+    public partial class HdfFile
```

Comment:
>HdfFile [](http://example.com/codeflow?start=25&length=7)

create a folder called HdfModel, add one file per class

Reply:
got it, will work on this.

Reply:
done.